Packs/CommonPlaybooks/Playbooks/playbook-Eradication_Plan.yml

```diff
@@ -187,12 +187,6 @@ tasks:
             value:
               simple: "False"
           ignorecase: true
```

Comment:
Is the operator deletion was on purpose?

Reply:
Yes, it contained the non existing playbook input and is not needed for this condition so it was removed